doc/components_2.md

```diff
@@ -25,23 +22,40 @@ You can wrap your external mjml files inside the default `mjml > mj-body`
 </mjml>
 ```
 
+The `main.mjml` code here demonstrates adding content from `header.mjml`.
+
+The `header.mjml` code here is an include file without `mjml`, `mj-head`, and
+  `mj-body` tags.
+In this case, MJML puts all text from this file where
+  it finds the `mj-include`.
+
+MJML also allows the `mjml`, `mj-head`, and `mj-body` tags in include files.
+The resulting MJML include file can preview better outside the main template.
+When the main `mj-body` section holds these mixed include files,
+  MJML puts content of
+  (1) the `mj-head` from `mj-include` in the main `mj-head` element and 
+  (2) the `mj-body` from `mj-include` where it finds the `mj-include`.
+
+Include files can have `mj-include` elements.
```

Comment:
I'm fine about adding a new introduction about `mj-include`, but here it's way too much text.

2 things need to be documented here : 
- Use of include file can be direct (without mjml root tag) or a complete MJML document
- Included file can use mj-include too and the path will be relative to the included file.

Reply:
> and the path will be relative to the included file. 

Removing that part as it common sense to work like that.